client/src/routes/home/index.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
```

Comment:
Rename the route folder to `dashboard`, as well as the chunk and everything related